controllers/ingestendpoint/secret.go

```diff
@@ -111,8 +115,53 @@ func (g *EndpointSecretGenerator) PrepareFields(ctx context.Context, dk *dynatra
 		dataIngestToken = string(token)
 	}
 
+	diUrl, err := getDataIngestUrlFromDk(dk)
```

Comment:
small nitpick:
rename to `dataIngestUrl` so its clearer (also its more consistent with other names such as `dataIngestToken`)

Reply:
Functions renamed and small wrapper added so I hope it is clearer now

Reply:
actually I meant you to just rename `diUrl` to `dataIngestUrl`, but this is nicer :joy: